app/controllers/vm_common.rb

```diff
@@ -1324,13 +1324,15 @@ def replace_right_cell(options = {})
         # these subviews use angular, so they need to use a special partial
         # so the form buttons on the outer frame can be updated.
         if @sb[:action] == 'dialog_provision'
-          presenter.update(:form_buttons_div, r[
-            :partial => 'layouts/x_dialog_buttons',
-            :locals  => {
-              :action_url => action,
-              :record_id  => @edit[:rec_id],
-            }
-          ])
+          if Settings.product.old_dialog_user_ui
```

Comment:
There is similar code in `app/controllers/mixins/custom_button_dialog_form_mixin.rb#L9-L16` that could possibly be affecting other areas (Infra Networking), but I was not able to duplicate the problem. Because of that assumption that this doesn't fully solve the issue - the PR is WIP.